test/functional/tests/fault_injection/test_max_io_greater_in_core.py

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+import pytest
+
+from api.cas import casadm, cas_module
+from api.cas.cache_config import CacheMode, CacheLineSize
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+from test_tools.dd import Dd
+from test_utils import os_utils
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.parametrizex("module", cas_module.CasModule)
+@pytest.mark.parametrizex("cache_mode", CacheMode)
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.require_plugin("scsi_debug", delay="0", virtual_gb="4", dev_size_mb="500",
+                            sector_size="512", physblk_exp="4")
+def test_max_io_greater_in_core(module, cache_mode, cache_line_size):
+    """
+        title: Test behavior when core's max IO (max_sectors_kb) is greater than cache's.
+        description: |
+          Test behavior when core's max IO (max_sectors_kb) is greater than cache' for
+          various CAS configurations.
+        pass_criteria:
+          - No kernel bug.
+          - Running workload successfully.
+          - Fulfilling criteria: core.max_io > cache.max_io & core.max_io > 1MiB.
```

Comment:
Is this pass criterion or rather execution criterion?

Reply:
we can fail test if this is not true so I thought I can place it here. But I can just remove it, no problem :) I will follow your suggestion 😎

Reply:
You better ask one of our validation experts.